### PR TITLE
Boilerplater Files

### PR DESCRIPTION
**Pull request by perploug**
- adds code of conduct
- adds contribution file
- adds security file

Make sure to update `TODO` parts

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+### TODO
+
+## Workflow
+
+We follow the [GitHub Flow Workflow](https://guides.github.com/introduction/flow/)
+
+### TODO: Below is an Example
+
+1.  Fork the project 
+1.  Check out the `master` branch
```
Comment (herder) [inline CONTRIBUTING.md:24]:
```suggestion
1.  Check out the `main` branch 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+We follow the [GitHub Flow Workflow](https://guides.github.com/introduction/flow/)
+
+### TODO: Below is an Example
+
+1.  Fork the project 
+1.  Check out the `master` branch 
+1.  Create a feature branch
+1.  Write code and tests for your change 
+1.  From your branch, make a pull request against `https://ghe.spotify.net/sebastianq/find-candidates/master`
```
Comment (herder) [inline CONTRIBUTING.md:27]:
```suggestion
1.  From your branch, make a pull request against `https://ghe.spotify.net/spotify/fmt-maven-plugin/main` 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+### TODO: Below is an Example
+
+1.  Fork the project 
+1.  Check out the `master` branch 
+1.  Create a feature branch
+1.  Write code and tests for your change 
+1.  From your branch, make a pull request against `https://ghe.spotify.net/sebastianq/find-candidates/master` 
+1.  Work with repo maintainers to get your change reviewed 
+1.  Wait for your change to be pulled into `https://ghe.spotify.net/sebastianq/find-candidates/master`
```
Comment (herder) [inline CONTRIBUTING.md:29]:
```suggestion
1.  Wait for your change to be pulled into `https://ghe.spotify.net/spotify/fmt-maven-plugin/main`
```

**Review by herder — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## Workflow
+
+We follow the [GitHub Flow Workflow](https://guides.github.com/introduction/flow/)
+
+1.  Fork the project 
+1.  Check out the `main` branch 
+1.  Create a feature branch
+1.  Write code and tests for your change 
+1.  From your branch, make a pull request against `https://ghe.spotify.net/spotify/fmt-maven-plugin/main`
```
Comment (herder) [inline CONTRIBUTING.md:19]:
```suggestion
1.  From your branch, make a pull request against `https://github.com/spotify/fmt-maven-plugin/main` 
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+We follow the [GitHub Flow Workflow](https://guides.github.com/introduction/flow/)
+
+1.  Fork the project 
+1.  Check out the `main` branch 
+1.  Create a feature branch
+1.  Write code and tests for your change 
+1.  From your branch, make a pull request against `https://ghe.spotify.net/spotify/fmt-maven-plugin/main` 
+1.  Work with repo maintainers to get your change reviewed 
+1.  Wait for your change to be pulled into `https://ghe.spotify.net/spotify/fmt-maven-plugin/main`
```
Comment (herder) [inline CONTRIBUTING.md:21]:
```suggestion
1.  Wait for your change to be pulled into `https://github.com/spotify/fmt-maven-plugin/main`
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+## Code Review
+
+Branch protection is set up to require one approving review from maintainers of this repo. 
+
+## Documentation
+
+See [README](README.md)
+
+We also welcome improvements to the project documentation or to the existing
+docs. Please file an [issue](https://$REPOURL/issues/New).
```
Comment (herder) [inline CONTRIBUTING.md:64]:
```suggestion
docs. Please file an [issue](https://github.com/spotify/fmt-maven-plugin/issues/New).
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+# License 
+
+By contributing your code, you agree to license your contribution under the 
+terms of the [LICENSE](https://$LINKTOLICENSEFILE)
```
Comment (herder) [inline CONTRIBUTING.md:69]:
```suggestion
terms of the [LICENSE](LICENSE)
```

**Review by herder — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (klaraward) [reply CONTRIBUTING.md:19]: Good catch :)

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+- Your contact information.
+- Names (real, nicknames, or pseudonyms) of any individuals involved. If there are additional witnesses, please
+include them as well. Your account of what occurred, and if you believe the incident is ongoing. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger), please include a link.
+- Any additional information that may be helpful.
+
+After filing a report, a representative will contact you personally, review the incident, follow up with any additional questions, and make a decision as to how to respond. If the person who is harassing you is part of the response team, they will recuse themselves from handling your incident. If the complaint originates from a member of the response team, it will be handled by a different member of the response team. We will respect confidentiality requests for the purpose of protecting victims of abuse.
+
+### Attribution & Acknowledgements
+
+We all stand on the shoulders of giants across many open source communities.  We'd like to thank the communities and projects that established code of conducts and diversity statements as our inspiration:
```
Comment (herder) [inline CODE_OF_CONDUCT.md:63]: I think this looks more natural?

```suggestion
We all stand on the shoulders of giants across many open source communities.  We'd like to thank the communities and projects that established codes of conduct and diversity statements as our inspiration:
```

**Review by herder — APPROVED** [review]

**Review by caesar-ralf — APPROVED** [review]
LGTM

File: CODE_OF_CONDUCT.md (hunk 8 above)
Comment (klaraward) [reply CODE_OF_CONDUCT.md:63]: 21M google hits for "codes of conduct", 200K for "code of conducts" so yes. This text comes from https://ghe.spotify.net/foss/repo-files/blob/master/CODE_OF_CONDUCT.md, so maybe we can update there as well?

Comment (herder) [reply CODE_OF_CONDUCT.md]: Created a PR for it, @klaraward

**Pull request merged**